src/dynimage.rs

```diff
@@ -324,6 +302,25 @@ impl DynamicImage {
         }
     }
 
+    fn resize_dimensions(&self, nwidth: u32, nheight: u32, fill: bool) -> (u32, u32) {
```

Comment:
I think it would be beneficial to have a unit test for this function. However, I'm not sure where it should go. There aren't many tests written at the bottom of this file.

Reply:
The bottom at the file is fine, I think. Could put it here: https://github.com/PistonDevelopers/image/blob/master/src/dynimage.rs#L764